### DISCUSSION TITLE
Create the codemagic.yaml file with the options done in the Workflow-Editor

### DISCUSSION THREAD

**Discussion by gliden** [codemagic-ci-cd/codemagic-docs#2271] [Ideas]
### Use case

```Text
The Workflow-Editor is an easy tool to create a build setting. 
We have multiple apps with nearly the same settings an some advanced settings.
```

### Idea

```Text
It would be nice to have the opportunity to create the codemagic.yaml file based on the settings done in the workflow editor.
```

**Comment by Ycoi** [comment]
Thanks for the feedback @gliden! Actually, in the long term we are looking into moving to improved yaml onboarding and visual yaml editor instead of the current workflow editor. So in the long term we will get to the position youre describing.

However, especially since you have several apps with nearly the same settings, switching to the yaml already today might be a good option for you. That way when adding a new app you only have to copy the existing yaml of an old app and then modifying the few advanced settings. If this is something you want to do, I recommend checking our [guides](https://docs.codemagic.io/yaml-quick-start/building-a-flutter-app/) and [sample projects](https://docs.codemagic.io/yaml-quick-start/codemagic-sample-projects/) for details. 

And if you have issues with creating your yaml file, you can always ask help in the [Q&A section here in GitHub Discussions](https://github.com/orgs/codemagic-ci-cd/discussions/categories/q-a) or join our [Discord community](https://codemagic.io/discord/).
